src/components/home/Landing.jsx

```diff
@@ -1,5 +1,53 @@
+import Image from "next/image";
+import logo from "@/public/prishi-logo.webp";
+import landing from "@/public/landing.webp";
+import Button from "@/components/Button";
+import Link from "next/link";
+
 const Landing = () => {
-  return <div>Landing</div>;
+  return (
+    <div className="flex h-screen w-screen ">
+      <div className="flex flex-col w-7/12 justify-center gap-3 pl-12">
```

Comment:
u can honestly make it w-8/12 so the largest text doesn't wrap until smaller screen sizes